tests/integration/test_interfaces.py

```diff
@@ -57,7 +57,7 @@ def test_bridge_with_vlan(lldpd1, lldpd, lldpcli, namespaces, links, when):
         out = lldpcli("-f", "keyvalue", "show", "neighbors", "details")
         assert out['lldp.eth0.port.descr'] == 'eth1'
         assert out['lldp.eth0.vlan'] == \
-            ['vlan100', 'vlan200', 'vlan300']
+            ['vlan-100', 'vlan-200', 'vlan-300']
```

Comment:
This is a classic bridge with VLAN on top of it. The name of interfaces are `vlan100`, ...